mithril-aggregator/src/database/record/certificate_pending.rs

```diff
@@ -0,0 +1,85 @@
+use chrono::{DateTime, Utc};
+use mithril_common::entities::{CertificatePending, Epoch};
+use mithril_persistence::sqlite::{HydrationError, Projection, SqLiteEntity};
+
+/// CertificatePending record is the representation of a stored pending certificate.
+pub struct CertificatePendingRecord {
+    /// Current Epoch
+    pub epoch: Epoch,
+
+    /// Pending certificate serialization as json
+    pub certificate: String,
+
+    /// Date and time when the pending certificate was created
+    pub created_at: DateTime<Utc>,
+}
+
+impl CertificatePendingRecord {
+    /// Construct a [Projection] that will allow to hydrate this `CertificatePendingRecord`.
+    pub fn get_projection_with_table(table: &str) -> Projection {
```

Comment:
It's a proposition to have a function that produce projection with that final table name instead of using an alias.
Tell me if we put it in this PR or if you prefer to keep the standard way

The alias mechanism is not really necessary because we always calling it just after creating the projection. There is no reuse of the projection.
With this function, the caller knows which table name(s) need to be map instead of using an alias key string without any constraints and used in different files (we may use a constant).
Callers of projection function are requests that know record type and can use specific function of this record.